src/app/dashboard/filter/label-order.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'labelOrder'
+})
+export class LabelOrderPipe implements PipeTransform {
+  transform(options: { [label: string]: string }[], label: string) {
+    if (!options) {
+      return [];
+    }
+    return options.sort((a, b) => a[label].localeCompare(b[label]));
```

Comment:
I didn't even know `localCompare` was a thing!  Slick.

Reply:
for strings!

also should have noted how Angular mentions why they don't provide a sorting pipe: https://angular.io/guide/pipes#appendix-no-filterpipe-or-orderbypipe